Directory.Build.props

```diff
@@ -3,6 +3,7 @@
     <HighEntropyVA>true</HighEntropyVA>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <TargetFramework>net6.0</TargetFramework>
+    <NoWarn>NU1605</NoWarn>
```

Comment:
This doesn't seem like something we should be ignoring. Can we not fix whatever the downgrade being flagged is? Or if there's a reason why this is OK then at the least add a comment explaining why this needs to be done.